website/docs/install-config/configuration/configuration.mdx

```diff
@@ -65,41 +65,76 @@ To check if your config has been applied correctly, you can run the following co
 
 ## PostgreSQL Settings
 
-- `AUTHENTIK_POSTGRESQL__HOST`: Hostname of your PostgreSQL Server
-- `AUTHENTIK_POSTGRESQL__NAME`: Database name
-- `AUTHENTIK_POSTGRESQL__USER`: Database user
-- `AUTHENTIK_POSTGRESQL__PORT`: Database port, defaults to 5432
-- `AUTHENTIK_POSTGRESQL__PASSWORD`: Database password, defaults to the environment variable `POSTGRES_PASSWORD`
+### Basic Connection Settings
+
+- `AUTHENTIK_POSTGRESQL__HOST`: Defines the hostname or IP address of your PostgreSQL server where authentik will store its data.
+- `AUTHENTIK_POSTGRESQL__NAME`: Specifies the name of the database that authentik will use within PostgreSQL.
+- `AUTHENTIK_POSTGRESQL__USER`: Defines the username that authentik will use to authenticate with PostgreSQL.
+- `AUTHENTIK_POSTGRESQL__PORT`: Specifies the port on which PostgreSQL is listening. Defaults to the standard PostgreSQL port `5432`.
+- `AUTHENTIK_POSTGRESQL__PASSWORD`: Defines the password that authentik will use to authenticate with PostgreSQL. If not specified, defaults to the value of the environment variable `POSTGRES_PASSWORD`. Note that this default behavior only applies when using the default Docker Compose configuration.
```

Comment:
let's have this order:
- host
- port
- user
- password
- name
- schema_name